components/blogger/blogger.app.mjs

```diff
@@ -1,11 +1,135 @@
+import { axios } from "@pipedream/platform";
+
 export default {
   type: "app",
   app: "blogger",
-  propDefinitions: {},
+  propDefinitions: {
+    blog: {
+      type: "string",
+      label: "Blog",
+      description: "Blog where the action will be performed.",
+      async options() {
+        const data = await this.getUserBlogs();
+        return data.items.map((blog) => ({
+          label: blog.name,
+          value: blog.id,
+        }));
+      },
+    },
+    post: {
+      type: "string",
+      label: "Post",
+      description: "Post where the action will be performed.",
+      async options({
+        blog,
+        status,
+        prevContext,
+      }) {
+        const data = await this.getBlogPosts(
+          this,
+          blog,
+          status,
+          prevContext.nextPageToken,
+        );
+
+        return {
+          options: data.items.map((item) => ({
+            label: item.title,
+            value: item.id,
+          })),
+          context: {
+            nextPageToken: data.nextPageToken,
+          },
+        };
+      },
+    },
+    title: {
+      type: "string",
+      label: "Title",
+      description: "Your post title.",
+    },
+    content: {
+      type: "string",
+      label: "Content",
+      description: "Your post content.",
+    },
+  },
   methods: {
-    // this.$auth contains connected account data
-    authKeys() {
-      console.log(Object.keys(this.$auth));
+    _getBaseUrl() {
+      return "https://www.googleapis.com/blogger/v3";
+    },
+    _getHeaders() {
+      return {
+        "Content-Type": "application/json",
+        "Authorization": `Bearer ${this.$auth.oauth_access_token}`,
+      };
+    },
+    _getAxiosParams(opts = {}) {
+      return {
+        ...opts,
+        url: this._getBaseUrl() + opts.path,
+        headers: this._getHeaders(),
+      };
+    },
```

Comment:
I was recently told in a review that we should call methods like these `getRequestParams` instead of axios so its intended usage can be understood by someone who doesn't know what axios is
```suggestion
    _getRequestParams(opts = {}) {
      return {
        ...opts,
        url: this._getBaseUrl() + opts.path,
        headers: this._getHeaders(),
      };
    },
```